src/plugins/crypto.c

```diff
@@ -396,8 +446,32 @@ static gboolean luks_format (const gchar *device, const gchar *cipher, guint64 k
         }
     }
 
-    ret = crypt_format (cd, CRYPT_LUKS1, cipher_specs[0], cipher_specs[1],
-                        NULL, NULL, key_size, NULL);
+    if (extra) {
```

Comment:
I think there should be a block before this that checks that no LUKS2-only extra parameters were specified for LUKS1.

Reply:
Fixed.